pkg/engine/plan.go

```diff
@@ -223,6 +225,18 @@ func (res *planResult) Walk(ctx *Context, events deploy.Events, preview bool) (d
 				return
 			}
 
+			// Warn the user if they're not updating a resource whose initialization failed.
+			if step.Op() == deploy.OpSame && len(step.Old().InitErrors) > 0 {
+				prefix := colors.SpecImportant + "         [ERROR]" + colors.Reset
```

Comment:
can you add a screenshot to the description of what this looks like both during update and when update is complete.  i'd like to understand why we need to do things like add this whitespace here.

Reply:
Sure:

<img width="688" alt="screen shot 2018-07-20 at 1 29 00 pm" src="https://user-images.githubusercontent.com/1409156/43023862-239d0192-8c21-11e8-9079-bfc560353abc.png">

Reply:
ok.  so my preference would be to not add the whitespace, but have that done at the presentation alyer.  i.e. if we have multiple lines, align them after the "error/warning/info<colon>" text.

Reply:
This is minor though.  So i'm not going to gate the PR on this.  However, this sort of ting does impact the ability to change how the CLI works.  ON hte other hand, if the data is just a long string with newlines, the CLI can decide how to present it (including how ot indent multi-line constructs).

Reply:
Ok, we agreed offline to move that to a different review.